internal/configs/transportserver.go

```diff
@@ -16,6 +17,25 @@ type TransportServerEx struct {
 	PodsByIP        map[string]string
 }
 
+// TransportServerConfigurator generates a transportServer configuration
+type transportServerConfigurator struct {
+	isPlus         bool
+	enableSnippets bool
+	warnings       Warnings
+}
+
+// newTransportServerConfigurator creates a new TransportServerConfigurator
+func newTransportServerConfigurator(
+	isPlus bool,
+	staticParams *StaticConfigParams,
+) *transportServerConfigurator {
+	return &transportServerConfigurator{
+		isPlus:         isPlus,
+		enableSnippets: staticParams.EnableSnippets,
+		warnings:       make(map[runtime.Object][]string),
```

Comment:
why not `newWarnings()` like in `transportserver_test.go` ?

Reply:
Done